Lib/test/test_class.py

```diff
@@ -503,6 +503,58 @@ def __eq__(self, other): return 1
 
         self.assertRaises(TypeError, hash, C2())
 
+    def testPredefinedAttrs(self):
+        o = object()
+
+        class Custom:
+            pass
+
+        c = Custom()
+
+        methods = (
+            "__new__", "__init__",
+            "__repr__", "__str__", "__format__",
+            "__eq__", "__ne__", "__lt__", "__le__", "__gt__", "__ge__",
+            "__hash__",
+            "__getattribute__", "__setattr__", "__delattr__", "__dir__",
+        )
+        for name in methods:
+            with self.subTest(name):
+                self.assertTrue(callable(getattr(object, name)))
```

Comment:
The 2-arg form of getattr raises for non-attributes.  These calls need 3rd arg None.  `callable(None)` is `False`.

Reply:
Either way, if the attribute was missing, that would fail the test. Either because None is not callable, or because _getattr_ cannot find the attribute. Do you think adding None would be clearer?

Reply:
There would be a test error (unexpected exception), not a test failure (a test was tried and did not give the expected answer)